lms/templates/learner_dashboard/upgrade_message_subscription.underscore

```diff
@@ -0,0 +1,21 @@
+<div class="message certificate-status col-12 md-col-8">
+    <span class="card-msg"><%- gettext('Certificate Status') %></span>
+    <div><%- gettext('Needs verified certificate ') %></div>
```

Comment:
nit: is there a reason this is a `div` while the `upgrade_message.underscore` has it as a `span`? Will there be any unintentional CSS changes?

Reply:
so in the existing flow, the verified text is at the right hence they are both span
<img width="846" alt="image" src="https://user-images.githubusercontent.com/111358247/236414336-5c7391fe-ee6f-448a-aef0-d3f915582a6f.png">


but in the subscription design depending on the subState, its either this
<img width="355" alt="image" src="https://user-images.githubusercontent.com/111358247/236414205-77e98938-da2f-4504-8478-11df0649c7b1.png">
or this
<img width="352" alt="image" src="https://user-images.githubusercontent.com/111358247/236414804-90b0253e-8d61-4264-8bef-09caf351917c.png">

so the div is there to make the text be on the second line,
I will reconfirm with the design if that was intentional or not

Reply:
We confirmed that it was an over-sight and has made the div to span change in code